landing-page/components/Footer.tsx

```diff
@@ -68,7 +92,7 @@ const Footer = () => {
         {linkGroups.map((group, index) => (
           <div key={index} className={tw('mobile:w-full desktop:w-[192px] mobile:text-center')}>
             {Object.entries(group).map(([title, links]) => (
-              <FooterLinkGroup key={title} title={title} links={links} />
+              <FooterLinkGroup key={title} title={translation[title.toLowerCase() as keyof typeof translation] } links={links} />
```

Comment:
Please add a Todo here. The technical `.toLowerCase()` on the key is weird. There must be an alternative ...